src/main.rs

```diff
@@ -270,13 +268,13 @@ fn handle_docker_cmds(
         let xs = if a.is_present("parameters") {
             a.values_of("parameters").unwrap().collect::<Vec<_>>()
         } else {
-            vec![]
+            Vec::new()
```

Comment:
these are equivalent, right?

Reply:
`Vec::new` can be given in places that expect a `FnOnce() -> Vec`, and is a little bit easier on the compiler as it doesn't need to evaluate the macro.

I think what I actually want to do is this:
```rust
let xs: Vec<&str> = a.values_of("parameters")
    .map_or_else(Vec::new, |params| params.collect());
```